source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -100,26 +123,122 @@ The ovirt-provider-ovn provides an opinionated subset of OpenStack's networking
+
+Also, only the **ipv6_address_mode** option will be implemented, since their
+behaviors kind of overlap - e.g. *slaac* and *dhcpv6-stateless* **require** RAs
+to configure their IP addresses, so, no point in having a separate setting to
+configure those. This behavior is consistent with OpenStack's Neutron
+implementation - it just forces both options to be equal when both are
+provided.
 
 ## Routing
 
 Routing among IPv6 subnets is seemlessly provided by an OVN logical router
 entity.
 
-Since the scope of the feature is just to implement stateful DHCPv6, no router
-changes are required.
+OVN logical routers are required whenever the 'slaac' or
+'dhcpv6-stateless' options are configured on the subnet, since they are what
+generates the RA messages that clients require to configure their IP addresses.
+
+They are also necessary when using the 'dhcpv6-stateful' option **if** the
+dhclient version running on the guest connecting to the network to which the
+subnet is attached assigns a /128 prefix - since the network prefix length must
+be configured via RA messages.
 
-If, in a given future, stateless address auto-configuration becomes a goal,
-the logical router ports have to be configured, setting its
-**ipv6_ra_configs:address_mode** to 'slaac'.
+Provisioning a router and configuring its interfaces should thus be performed,
+since it also automatically enables the guest VMs routes to be configured: it
+adds some complexity during the network / subnet setup stage, but eases the
+configuration of all VMs attached to the networks whose subnets are connected
+to a router.
+
+Just creating the logical router is indeed not enough; all subnets that require
+RA messages to be multicasted into the connected VMs **must be** connected to
+at least one OVN logical router.
+
+That will create OVN logical flows that are responsible for generating the
+Router Advertisements messages the clients require to configure their IP
+addresses.
+These RA messages are generated as replies to the Router Solicitation messages.
+
+Periodic RA messages are also required, to prevent the default route on the
+guest VM from expiring; it has a default value of 65536 seconds.
+Simply activating the periodic flag on the logical router ports is enough, since
+its min/max defaults are well below that +-18 hour interval.
+
+### Routing - implementation detail
 
 The above change would be implemented in the following way:
+
 1. when a subnet is created, it's **ipv6_address_mode** parameter is stored in
 the subnet's external ids
 2. when that subnet is attached to a router, its **ipv6_ra_configs:address_mode**
 should be set from the subnet's external ids address mode value.
 
+## Guest VM configuration
+
+There are three different ways to configure the guests, enabling them to have
+IPv6 addresses, each with different advantages / drawbacks.
+
+### DHCPv6
+
+DHCPv6 is one option to configure the guests and requires **dhclient** to be
+installed in the guest VM.
+
```

Comment:
No. In RHEL based distros NetworkManager can also use other tools.
Please note that Windows is a supported guest, too.

Reply:
Done